### PR TITLE
chore: update pyjwt

### PR DESCRIPTION
**Pull request by c0state**

### CODE REVIEW

File: flask_resty/jwt.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self, jwk_set, *args, **kwargs):
         self.jwk_set = jwk_set
 
     def decode(self, jwt, **kwargs):
-        unverified_header = self.get_unverified_header(jwt)
+        unverified_header = jwt_lib.get_unverified_header(jwt)
```
Comment (c0state) [inline flask_resty/jwt.py:75]: This got moved to a new `PyJWS` object which can be accessed directly via the `jwt` module

**Review by sloria — APPROVED** [review]
thanks for completing a 3-point ticket for us 😁 

Firefox 2022-02-07 at 15 01 28

**Comment by c0state** [comment]
Can someone also add @matt-mclaughlin-quantum-si as cloud team member also? Thanks.

**Comment by sloria** [comment]
invite sent to @matt-mclaughlin-quantum-si

**Comment by c0state** [comment]
This should be a v3 release correct because https://github.com/4Catalyzer/flask-resty/compare/v2.0.0...master#diff-be86804fe2ca36af9c77b587cb3b41811668560f76cef3acdf9730b5e6f25798R51 could be fundamentally different? If so, I'll push a `v3.0.0` tag?

**Comment by c0state** [comment]
> hm, i wouldn't normally count dependency upgrades as a breaking change so long as there's no public-facing API change, so I think this can be 2.1.0

https://github.com/4Catalyzer/flask-resty/compare/v2.0.0...master#diff-be86804fe2ca36af9c77b587cb3b41811668560f76cef3acdf9730b5e6f25798R51 returns the jwt module though which could have some breaking changes I think?

**Comment by sloria** [comment]
oh i see, i missed that. i suppose that's technically public API, yes. otoh, we at bfly aren't using `.pyjwt` anywhere. also, it'd be reasonable to say that only _documented_ API should be considered public.

**Comment by sloria** [comment]
but go ahead and bump up to v3 to be conservative

**Pull request merged**